modules/api_docs/help/api_docs.md

```diff
@@ -0,0 +1,13 @@
+# API documentation module
+
+This module use [Swagger-UI](https://swagger.io/tools/swagger-ui/) to present this LORIS instance modules API endpoints. By selecting a module name in the *Select  a spec* dropdown, it will provide a description of each of the selected module endpoints.
+
+Feel free to test the *Try it out* button for each endpoints.
```

Comment:
```suggestion
You can use the *Try it out* button to test each endpoint. (Note that this will send a real API request which may affect your data.)
```